### ISSUE TITLE
Issue with Compiled Appmenu for XFCE (GalliumOS)

### ISSUE THREAD

**Issue by rbreaves** [rilian-la-te/vala-panel-appmenu#108]
I am not sure what I am doing wrong but the plugin does not appear as an item that I can add to my panel. I am able to get it to compile though. The main issue I am trying to address has to do with Sublime Text and Thunar not showing their menus into the plugin from http://www.webupd8.org/2017/02/alternative-global-menu-for-mate-and.html, but I am not going to complain unless I have issues with the latest version compiled from source. Sadly it appears what I have compiled doesn't load at all, and I am not sure where I went wrong.

xfce version 4.12

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="GalliumOS 2.1"
NAME="GalliumOS"
VERSION="2.1 (Xenon)"
ID=galliumos
ID_LIKE="ubuntu debian"
PRETTY_NAME="GalliumOS 2.1"
VERSION_ID="2.1"
HOME_URL="https://galliumos.org/"
SUPPORT_URL="https://galliumos.org/"
BUG_REPORT_URL="https://github.com/GalliumOS/galliumos-distro/issues"
VERSION_CODENAME=xenon
UBUNTU_CODENAME=xenial

```
user@gOS:~/Downloads/vala-panel-appmenu/build$ cmake .. -DENABLE_XFCE=ON -DENABLE_MATE=OFF -DENABLE_UNITY_GTK_MODULE=ON -DENABLE_JAYATANA=ON -DMAKE_BOLD_APPNAME=ON -DENABLE_BUDGIE=OFF -DENABLE_VALAPANEL=OFF
-- Looking for RPMTools... - found rpmuild is /usr/bin/rpmbuild
-- RPMTools:: Using RPM_ROOTDIR=/home/user/Downloads/vala-panel-appmenu/build/RPM
-- Configuring spec file </home/user/Downloads/vala-panel-appmenu/rpm/vala-panel-appmenu.spec.in> --> </home/user/Downloads/vala-panel-appmenu/build/RPM/SPECS/vala-panel-appmenu.spec>
-- RPMTools:: Using RPM_ROOTDIR=/home/user/Downloads/vala-panel-appmenu/build/RPM
-- Configuring spec file </home/user/Downloads/vala-panel-appmenu/rpm/appmenu-gtk-module.spec.in> --> </home/user/Downloads/vala-panel-appmenu/build/RPM/SPECS/appmenu-gtk-module.spec>
-- RPMTools:: Using RPM_ROOTDIR=/home/user/Downloads/vala-panel-appmenu/build/RPM
-- Configuring spec file </home/user/Downloads/vala-panel-appmenu/rpm/java-appmenu.spec.in> --> </home/user/Downloads/vala-panel-appmenu/build/RPM/SPECS/java-appmenu.spec>
-- checking for a minimum Vala version of 0.24
--   found Vala, version 0.34.9
-- Checking for module 'libwnck-3.0>=3.4.0'
--   
-- Checking for module 'vala-panel'
--   No package 'vala-panel' found
-- Checking for modules 'budgie-1.0;libpeas-1.0'
--   No package 'budgie-1.0' found
-- Checking for module 'gtk-doc>=1.26'
--   
-- GSettings schemas will be installed locally.
-- GSettings schemas will be compiled.
-- GSettings schemas will be installed into /usr/local/share/glib-2.0/schemas/
-- Could NOT find Java (missing:  Java_JAVA_EXECUTABLE Java_JAR_EXECUTABLE Java_JAVAC_EXECUTABLE Java_JAVAH_EXECUTABLE Java_JAVADOC_EXECUTABLE) 
-- Could NOT find JNI (missing:  JAVA_AWT_LIBRARY JAVA_JVM_LIBRARY JAVA_INCLUDE_PATH JAVA_INCLUDE_PATH2 JAVA_AWT_INCLUDE_PATH) 
-- Checking for module 'dbusmenu-glib-0.4'
--   No package 'dbusmenu-glib-0.4' found
-- checking for a minimum Vala version of 0.24
--   found Vala, version 0.34.9
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/Downloads/vala-panel-appmenu/build
user@gOS:~/Downloads/vala-panel-appmenu/build$ make
[ 15%] Built target appmenu-gtk3-parser
[ 31%] Built target appmenu-gtk2-parser
[ 39%] Built target appmenu-gtk2-module
[ 47%] Built target appmenu-gtk3-module
[ 57%] Built target vala-dbusmenu-o
[ 86%] Built target appmenu-core
[100%] Built target appmenu-xfce
[100%] Built target i18n
user@gOS:~/Downloads/vala-panel-appmenu/build$ sudo make install
[sudo] password for user: 
[ 15%] Built target appmenu-gtk3-parser
[ 31%] Built target appmenu-gtk2-parser
[ 39%] Built target appmenu-gtk2-module
[ 47%] Built target appmenu-gtk3-module
[ 57%] Built target vala-dbusmenu-o
[ 86%] Built target appmenu-core
[100%] Built target appmenu-xfce
[100%] Built target i18n
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/share/glib-2.0/schemas/org.valapanel.appmenu-gtk-module.gschema.xml
-- Compiling GSettings schemas
-- Installing: /usr/local/lib/pkgconfig/appmenu-gtk3-parser.pc
-- Installing: /usr/local/lib/pkgconfig/appmenu-gtk2-parser.pc
-- Installing: /usr/lib/systemd/user/appmenu-gtk-module.service
-- Installing: /usr/local/lib/libappmenu-gtk3-parser.so.0.5.2
-- Up-to-date: /usr/local/lib/libappmenu-gtk3-parser.so.0
-- Up-to-date: /usr/local/lib/libappmenu-gtk3-parser.so
-- Installing: /usr/local/lib/libappmenu-gtk2-parser.so.0.5.2
-- Up-to-date: /usr/local/lib/libappmenu-gtk2-parser.so.0
-- Up-to-date: /usr/local/lib/libappmenu-gtk2-parser.so
-- Installing: /usr/local/include/appmenu-gtk-module/unity-gtk-parser.h
-- Installing: /usr/local/include/appmenu-gtk-module/unity-gtk-menu-shell.h
-- Installing: /usr/local/include/appmenu-gtk-module/unity-gtk-action-group.h
-- Installing: /usr/lib/x86_64-linux-gnu/gtk-3.0/modules/libappmenu-gtk-module.so
-- Set runtime path of "/usr/lib/x86_64-linux-gnu/gtk-3.0/modules/libappmenu-gtk-module.so" to ""
-- Installing: /usr/lib/x86_64-linux-gnu/gtk-2.0/modules/libappmenu-gtk-module.so
-- Set runtime path of "/usr/lib/x86_64-linux-gnu/gtk-2.0/modules/libappmenu-gtk-module.so" to ""
-- Installing: /usr/local/lib/xfce4/panel/plugins/libappmenu-xfce.so
-- Installing: /usr/local/share/xfce4/panel/plugins/appmenu.desktop
-- Installing: /usr/local/share/locale/br/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/zh/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nb/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ak/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/gl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/uz/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/it/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ki/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ln/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ga/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ast/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/oj/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ii/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/dv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/he/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/as/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/st/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kj/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ta/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/su/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/to/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lt/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/gd/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bh/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/qu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/cs/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/de/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sa/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/om/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/hi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/zu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/si/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sm/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fr_CA/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sq/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/hu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/os/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ur/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/zh_HK/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/se/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/da/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/pl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/gv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ab/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/aa/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/te/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/la/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ts/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/az/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ff/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/yo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/vi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sd/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ny/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ae/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ha/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/gn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/wa/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ky/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/jv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/hz/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/rue/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/pt_BR/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ia/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ik/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/pa/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/rw/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/pi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/li/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/my/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/yi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/zh_CN/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lb/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ks/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/wo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/pt/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/eo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fa/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/am/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ko/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ba/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ss/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/en_AU/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/rm/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/oc/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ho/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/io/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/or/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/uk/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ig/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ng/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/za/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/th/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ms/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/is/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/es/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/xh/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/en_CA/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/gu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/el/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ee/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mh/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ty/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ru/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/eu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/an/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fy/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/cr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ne/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ve/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tl/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/rn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tw/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/dz/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/hr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fj/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ja/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kw/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bs/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ml/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/kk/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mt/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ar/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sc/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/cy/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bn/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/en_GB/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/af/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/so/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ckb/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sma/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bm/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/bi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/av/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/id/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sk/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/km/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/cu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mg/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ie/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/be/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ro/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mk/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/co/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/cv/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/lu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ku/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/vo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ht/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tk/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ay/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ti/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/fi/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/hy/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ce/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nd/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ca/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ka/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/mo/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/iu/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/sw/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/na/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ch/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ug/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/tt/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/nr/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/zh_TW/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/ps/LC_MESSAGES/vala-panel-appmenu.mo
-- Installing: /usr/local/share/locale/et/LC_MESSAGES/vala-panel-appmenu.mo
```

**Comment by rbreaves** [comment]
I found a newer build 0.4.4.1, but it also has the same menu issues with sublime and thunar.

https://blueprints.launchpad.net/~nilarimogard/+archive/ubuntu/test/+build/12420847

**Comment by rilian-la-te** [comment]
You should set CMAKE_INSTALL_PREFIX to /usr, AFAIK.

And you should use unity-gtk-module for Ubuntu.

**Comment by rbreaves** [comment]
Ah, well I must have overlooked that "set CMAKE_INSTALL_PREFIX to /usr". I believe you have mentioned that to others. As far as the " unity-gtk-module" I have installed what's available, but it is only "unity2-gtk-module" and " unity-gtk3-module" that I have been able to install, so I am not sure about " unity-gtk-module", it's just not part of any official repo on my system.

Regardless it does appear that installing the binaries from the nilarimogard repo and then logging out completely and logging back in has resolved my issues with all of my apps. It is solid now, thanks!

I will go ahead and close my comment since the binary and logging out and back in has resolved my issue.
